### PR TITLE
Jerry/more api documentation

### PR DESCRIPTION
**Pull request by jaywonder20**
Make sure you've checked off all these things before submitting:

- [ ] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes/_opportunities.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+```
+
+`POST https://api.apollo.io/v1/opportunities`
+
+| Parameter            | Description                                                                                                              |
+| -------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| owner_id             | Owner ID . You can GET a list of possible users and their associated information from [Misc/Users](#get-a-list-of-users) |
+| name                 | Opportunity Name                                                                                                         |
+| amount               | The Amount of money involved in the Opportunity/ Deal                                                                    |
+| opportunity_stage_id | The ID of the current Stage (e.g. Prospecting, Closed Won, Closed Lost) of this Opportunity                              |
```
Comment (draak-krijger) [inline source/includes/_opportunities.md:55]: Shouldn't we mention how `opportunity_stage_id` can be found(Similar to `owner_id`)?

File: source/includes/_opportunities.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+    "breadcrumbs": [],
+    "opportunities": [
+        {
+            "id": "5c14XXXXXXXXXXXXXXXXXXXX",
+            "team_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+            "owner_id": "5c100XXXXXXXXXXXXXXXXXXXX",
+            "salesforce_owner_id": "0051U000XXXXXXXXXX",
+            "amount": 90000.0,
+            "closed_date": "2018-08-27T15:00:00.000+00:00",
+            "account_id": "5c14223d945746a058358f93",
```
Comment (draak-krijger) [inline source/includes/_opportunities.md:80]: The standard example would be better,  like `5c14XXXXXXXXXXXXXX`.

File: source/includes/_opportunities.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+            "description": null,
+            "is_closed": true,
+            "is_won": true,
+            "name": "Apollo Inc",
+            "stage_name": "Closed Won",
+            "opportunity_stage_id": "5c10XXXXXXXXXXXXXXXXXXXX",
+            "typed_custom_fields": {
+                "5c1004a041f5ac0995d5f619": [],
+                "5c1004a041f5ac0995d5f61b": [
```
Comment (draak-krijger) [inline source/includes/_opportunities.md:89]: Avoiding the use of real id would be good.

File: source/includes/_opportunities.md [hunk 4]
```diff
@@ -0,0 +1,278 @@
...
+            "opportunity_stage_id": "5c14XXXXXXXXXXXXXXXXXXXX",
+            "typed_custom_fields": {},
+            "source": "crm",
+            "salesforce_id": "0061U000XXXXXXXXXX",
+            "created_at": "2020-01-16T00:41:30.834Z",
+            "existence_level": "full",
+            "opportunity_contact_roles": [],
+            "salesforce_record_url": "https://na85.salesforce.com/0061U000XXXXXXXXXX",
+            "account": {
+                "id": "5c14223d945746a058358f8f",
```
Comment (draak-krijger) [inline source/includes/_opportunities.md:153]: Avoiding the use of real id would be good.

File: source/includes/_opportunities.md [hunk 5]
```diff
@@ -0,0 +1,278 @@
...
+    "opportunity_contact_roles": [],
+    "salesforce_record_url": "https://na85.salesforce.com/0061U000XXXXXXXXXX",
+    "num_contacts": 0
+  }
+}
+```
+
+This endpoint returns the full information about an opportunity.
+
+`GET https://api.apollo.io/v1/opportunities/REPLACE_WITH_OPPORTUNITY_ID`
```
Comment (draak-krijger) [inline source/includes/_opportunities.md:230]: Providing an example of `REPLACE_WITH_OPPORTUNITY_ID` would be good. Similar to [this](https://github.com/apolloio/apollo-api-docs/pull/13/files#diff-6f010c6b732b5c0e0e87c8c3025000847791c8b7960ea16c7a79a513a4a943c1R52).

File: source/includes/_opportunities.md [hunk 6]
```diff
@@ -0,0 +1,278 @@
...
+}
+```
+
+Update the details of an already existing opportunity.
+
+<aside class="notice">
+Only supplied fields will be updated. All other fields will be left untouched.
+</aside>
+
+`PATCH https://api.apollo.io/v1/opportunities/REPLACE_WITH_OPPORTUNITY_ID`
```
Comment (draak-krijger) [inline source/includes/_opportunities.md:278]: Same here about `REPLACE_WITH_OPPORTUNITY_ID`

**Review by draak-krijger — APPROVED** [review]
Overall looks good to me!

Left some minor comment.

File: source/includes/_opportunities.md (hunk 1 above)
Comment (jaywonder20) [reply source/includes/_opportunities.md:55]: that's true but the opportunity_stages has no endpoint for viewing (only create and update endpoints are allowed)
and opportunity stages can also be gotten from the team object 

Screenshot 2020-12-23 at 19 29 40
Screenshot 2020-12-23 at 19 29 23

So possible solutions
1. an endpoint to get my team opportunities

File: source/includes/_opportunities.md (hunk 2 above)
Comment (jaywonder20) [reply source/includes/_opportunities.md:80]: oops I skipped that, I masked a whole lot of the id, my eyes skipped this

File: source/includes/_opportunities.md (hunk 1 above)
Comment (draak-krijger) [reply source/includes/_opportunities.md:55]: Try to mention that `opportunity_stage_id` is available at team object.

Comment (jaywonder20) [reply source/includes/_opportunities.md:55]: something like this right?
Screenshot 2020-12-23 at 19 51 42

Comment (draak-krijger) [reply source/includes/_opportunities.md:55]: Should be good.

**Pull request merged**